### DISCUSSION TITLE
Overriding CSS Frameworks.

### DISCUSSION THREAD

**Discussion by anshulnegitc** [getbem/getbem.github.io#237] [Q&A]
Often I come across the problem where I need to override install CSS framework classes so how can I achieve it through BEM.

**Comment by vacool6** [comment]
If you use any CSS frameworks you don't have to use BME cuz these frameworks can make use of any CSS class naming pattern internally. 

So we just need to follow the class names that they provide.

**Comment by JamesManningR** [comment]
I'm not 100% sure what you are saying but if I'm correct, your asking how you use BEM when using some form of ui-library

I've always stuck to this principal (if I choose to use BEM): BEM only applies to components that I have control over. Once you enter the UI library, it's within that UI libraries scope to define its own CSS. See your internal code as a layout engine for those components. Otherwise you're using a UI library which doesn't really suit your purposes, or BEM isn't applicable to that project.
